operator/src/main/resources/scripts/model_wdt_mii_filter.py

```diff
@@ -531,12 +531,12 @@ def customizeIstioReplicationChannel(server, name, listen_address):
     _writeIstioNAP(name='istiorepl', server=server, listen_address=listen_address,
                  listen_port=istio_repl_listen_port, protocol='t3', http_enabled="true",
                  bind_to_localhost="true", use_fast_serialization='true',
-                 tunneling_enabled='true')
+                 tunneling_enabled='false')
   else:
     _writeIstioNAP(name='istiorepl', server=server, listen_address=listen_address,
                    listen_port=istio_repl_listen_port, protocol='t3', http_enabled="true",
                    bind_to_localhost="false", use_fast_serialization='true',
-                   tunneling_enabled='true')
+                   tunneling_enabled='false')
```

Comment:
This file needs the copyright date updated

Reply:
done